runtime/src/commonMain/kotlin/com/squareup/sqldelight/Query.kt

```diff
@@ -78,32 +86,17 @@ private class SimpleQuery<out RowType : Any>(
  *   cursor returned by [execute] to [RowType].
  */
 abstract class Query<out RowType : Any>(
-  private val listeners: MutableList<Listener>,
   val mapper: (SqlCursor) -> RowType
 ) {
-  /**
-   * Notify listeners that their current result set is staled.
-   *
-   * Called internally by SQLDelight when it detects a possible staling of the result set. Emits
-   * some false positives but never misses a true positive.
-   */
-  fun notifyDataChanged() {
-    listeners.forEach(Listener::queryResultsChanged)
-  }
-
   /**
    * Register a listener to be notified of future changes in the result set.
    */
-  fun addListener(listener: Listener) {
```

Comment:
A sort of aside, but is this documented on the website at all? I can't seem to find it anywhere and I think it'd be useful for people to know about it (if for whatever reason they don't want to use RxJava or Kotlin Coroutines).

Reply:
only in the dokka stuff that gets generated, which might be enough but yea I could see adding more being important, especially since we're adding more public APIs in the driver about it might be worth having a section